api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpClientsTransitionedToDTGReport.java

```diff
@@ -178,7 +178,7 @@ public ReportDefinition constructReportDefinition() {
 
     @Override
     public String getVersion() {
-        return "3.0.2";
+        return "3.0.1";
```

Comment:
Why change from a higher version to a lower version

Reply:
@slubwama this has been resolved